src/main/java/software/sigma/internship/controller/FundController.java

```diff
@@ -0,0 +1,76 @@
+package software.sigma.internship.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springdoc.api.annotations.ParameterObject;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import software.sigma.internship.dto.FundDto;
+import software.sigma.internship.mapper.FundMapper;
+import software.sigma.internship.repository.FundRepository;
+import software.sigma.internship.service.FundService;
+
+@Tag(name = "Fund controller")
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/fund")
+public class FundController {
+    private final FundService fundService;
+    private final FundRepository fundRepository;
```

Comment:
In controller, you should only services.
Controllers work with services.
Services work with repositories and other services.